pkg/build/node/node.go

```diff
@@ -315,6 +315,9 @@ func (c *BuildContext) buildImage(dir string) error {
 		return err
 	}
 
+	// Let Bits clean up after itself.
+	c.bits.Clean()
```

Comment:
this won't run if we return early

Reply:
Hi @BenTheElder,

I considered that, and I thought that perhaps it would be desirable to leave the bits/data that failed to build for purposes of figuring out what went wrong...

Reply:
Seems reasonable for now. Might revisit later 👍